front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -159,13 +199,16 @@ public void bulkImport(Collection<T> items) {
    * Update local cache with any recently modified items.
    */
   protected void refresh() {
-    long startTime = System.nanoTime();
-    allItemsCache.set(fetchAllItems(allItemsCache.get()));
-    long elapsed = System.nanoTime() - startTime;
+    long startTime = System.currentTimeMillis();
 
-    registry.timer("storageServiceSupport.cacheRefreshTime", "objectType", objectType.name()).record(elapsed, TimeUnit.NANOSECONDS);
+    cacheRefreshTimer.record(new Runnable() {
+      public void run() {
+          allItemsCache.set(fetchAllItems(allItemsCache.get()));
```

Comment:
Originally I moved it because I was using datadog and didnt see the objectType metric anymore so thought this API might not have been working right, but then when it still didnt work, I realized it was because datadog converts mixed case labels to lower case labels so that's why it wasnt seeing objectType (because it converted it to objecttype). I then left it because it was consistent with everything else. I copied them in from a pending [working] PR I had before you submitted this (and had used a different label so didnt notice the mixed case thing).

I'll change all my uses into the pattern you had. I didnt realize this was even a usage pattern originally.